provision/README.md

```diff
@@ -120,6 +120,12 @@ In the case that provision service is not deployed with credentials or API key o
 curl -s -S  -X POST  http://localhost:8091/mapping -H "Authorization: <token|api_key>" -H 'Content-Type: application/json' -d '{ "external_id" : "<external_id>", "external_key":"<external_key>" }'
 ```
 
+Or if you want to specify a name for thing different than in `config.toml` you can specify post data as:
+
+```json
+{ "name":"<name>","external_id" : "<external_id>", "external_key":"<external_key>" }
```

Comment:
Add white space only after `:` and `,`: `{"name": "<name>", "external_id": "<external_id>", "external_key": "<external_key>"}`

Reply:
Yes, this needs uniform representation.